docs/ManualDeploymentGuideScalarDBServerOnAKS.md

```diff
@@ -1,72 +1,55 @@
 
-Deploy Scalar DB Server on your AKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
+Note: When you deploy your application on other environments than the AKS cluster that you deploy ScalarDB Server, you have to set the `LoadBalancer` to the `envoy.service.type` parameter to access Scalar Envoy from your application.
 
-* [Deploy Scalar Products using Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+## Step 6. Deploy ScalarDB Server using Scalar Helm Chart
 
-## Step 7. Check the status of Scalar DB Server deployment
+Deploy ScalarDB Server on your AKS cluster using Scalar Helm Chart. See [Deploy Scalar Products using Scalar Helm Chart](https://github.com/scalar-labs/helm-charts/blob/main/docs/how-to-deploy-scalar-products.md) for more details.
 
-After deploying Scalar DB Server on your AKS cluster, you need to check the status of each component. Please refer to the following document for more details.  
+Note: We recommend creating a dedicated namespace by using the `kubectl create ns scalardb` command and deploying ScalarDB Server in the namespace by using the `-n scalardb` option with the `helm install` command.
 
-* [What you might want to check on a regular basis](./RegularCheck.md) // TODO: Update existing document
+## Step 7. Check the status of ScalarDB Server deployment
 
-## Step 8. Monitoring for Scalar DB Server deployment
+After deploying ScalarDB Server on your AKS cluster, you need to check the status of each component. See [What you might want to check on a regular basis](./RegularCheck.md) for more details.
 
-After deploying Scalar DB Server on your AKS cluster, it is recommended to monitor the deployed components and collect their logs, especially in production. Please refer to the following document for more details.  
+## Step 8. Monitoring for ScalarDB Server deployment
 
-* [Kubernetes Monitor Guide](./K8sMonitorGuide.md) // TODO: Update existing document
-* [How to collect logs from Kubernetes applications](./K8sLogCollectionGuide.md) // TODO: Update existing document
+After deploying ScalarDB Server on your AKS cluster, we recommend monitoring the deployed components and collecting their logs, especially in production. See [Monitoring Scalar products on a Kubernetes cluster](./K8sMonitorGuide.md) and [Collecting logs from Scalar products on a Kubernetes cluster](./K8sLogCollectionGuide.md) for more details.
 
 ---
 
-## Uninstall Scalar DB Server on AKS
+## Uninstall ScalarDB Server on AKS
 
-If you want to uninstall the environment you created, please uninstall/remove resources in the reverse order of creation.  // TODO: Add delete steps in each document
```

Comment:
Actually, I didn't create concrete deletion steps in each document. However, I think the priority of deleting steps is low at this time. So, I will create concrete deleting steps in another task if we need in the future.